### DISCUSSION TITLE
What is the easiest way to remove the ActionButtons bar at the bottom

### DISCUSSION THREAD

**Discussion by ghost** [gsantner/markor#2347] [Q&A] [answered]
![Screenshot_20240714_013641](https://github.com/user-attachments/assets/83da4029-eb50-4803-9692-d8e83963c5aa)

What is the piece of code which if I commented out the bottom bar would stop showing completely? Which piece of code adds it to the edit view?

Thanks.

**Comment by harshad1** [comment, marked as answer]
Find this id and set the view visibility to gone should do it I think

```
        android:id="@+id/document__fragment__edit__text_actions_bar__scrolling_parent"
```

**Reply by ghost** [reply]
Thanks very much.

**Comment by harshad1** [comment]
I added code to do this in my pr, btw:

https://github.com/gsantner/markor/pull/2307/commits/167d5e9e42481ff7cfde4ebd803b2b2cd428f0b6#diff-a5f18ed8d9a7afd47e78969d79f34ea509f949c4f383113680d828316380943dR675

If you disable all the action buttons it will now hide the action bar.

**Reply by ghost** [reply]
Beautiful!
